crates/codegen/parser/runtime/src/napi/napi_cursor.rs

```diff
@@ -33,6 +33,7 @@ impl Cursor {
     }
 
     #[napi]
+    #[allow(clippy::should_implement_trait)] // These are meant to be explicitly exposed to NAPI
```

Comment:
I wonder if we are able to move this to an explicit `#[napi] impl Clone for Cursor`, not just to appease this lint, but just in case we actually need to clone it in another NAPI util/containing type here.